### PR TITLE
Add redirect from .com to .org

### PR DESCRIPTION
**Pull request by wgreenberg**

### CODE REVIEW

File: nginx.conf [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
+    server_name privacybadger.com www.privacybadger.com;
+
+    location / {
+        return 301 https://privacybadger.org$request_uri;
+    }
+}
+
+server {
+    listen 80;
+    server_name privacybadger.org www.privacybadger.org localhost;
```
Comment (ghostwords) [inline nginx.conf:12]: Should we also redirect the www subdomain?

Comment (wioux) [reply nginx.conf:12]: I think so. Having content duplicated on multiple domains like this can cause problems with search rankings.

Comment (wgreenberg) [reply nginx.conf:12]: to clarify, you're saying www.privacybadger.org should redirect to privacybadger.org?

Comment (ghostwords) [reply nginx.conf:12]: Yes. Isn't it the same issue? We want one canonical domain only.

Comment (wgreenberg) [reply nginx.conf]: sounds good, updated the pr

**Comment by ghostwords** [comment]
Is this waiting on anything else? Will merging and redeploying the website apply this fix?

**Comment by wgreenberg** [comment]
Nope, it’s just waiting on a merge and redeploy!

**Comment by ghostwords** [comment]
Redirects are live. Thank you!

**Pull request merged**